internal/pkg/form/form.go

```diff
@@ -112,6 +113,19 @@ func ConfirmDeletion(cmd *cobra.Command, promptMsg, stringToType string) (bool,
 	return false, errors.NewErrorWithSuggestions(fmt.Sprintf(`input does not match "%s"`, stringToType), DeleteResourceConfirmSuggestions)
 }
 
+func ConfirmEnter(cmd *cobra.Command) error {
+	// This function prevents echoing of user input instead of displaying text or *'s
+	// so that the CLI will appear to wait until 'enter' or 'Ctrl-C' are entered.
+	utils.Print(cmd, "Press enter to continue or Ctrl-C to cancel:")
```

Comment:
Would this prevent us from needing the second `Print()` below?
```suggestion
	utils.Println(cmd, "Press enter to continue or Ctrl-C to cancel:")
```

Reply:
No; the second print is there for an unrelated reason. I just added a comment to explain it.